rust/ql/lib/codeql/rust/controlflow/ControlFlowGraph.qll

```diff
@@ -9,6 +9,8 @@ private import BasicBlocks
 
 final class CfgScope = Scope::CfgScope;
 
+predicate getEnclosingCfgScope = Scope::getEnclosingCfgScope/1;
```

Comment:
We rarely expose classless predicates like this one; I think it would be better as a member predicate on `AstNode`, like `getEnclosingCallable`. We probably also want to cache it like with `AsNode.getEnclosingCallable`.